.github/workflows/validate-shell-scripts.yml

```diff
@@ -1,5 +1,5 @@
 ---
-name: Validate shell scripts
+name: Validate shell scripts - Ubuntu and MacOS
```

Comment:
Revert this change so we don't need to keep this up to date when adding new OSes

Reply:
Reverted